src/docs/index.md

```diff
@@ -32,42 +32,20 @@ you are looking for. Some of the changes to the website include:
 
 Some of the new content includes:
 
-* Additional [cookbook](/docs/cookbook) recipes
 * Deep dive on Flutter internals,
   [Inside Flutter](/docs/resources/inside-flutter)
 * [Technical videos](/docs/resources/videos) page
 * How to connect [a native debugger _and_
   a Dart debugger to your app](/docs/testing/oem-debuggers)
-  (not yet complete)
+  (iOS info in process)
```

Comment:
What does "iOS info in process" mean?

Reply:
It's only done for Android. I will be changing before launch.

Reply:
Maybe it should read something like "iOS support in progress"?